### ISSUE TITLE
Ability to configure axios timeout

### ISSUE THREAD

**Issue by gaving** [searchkit/searchkit#345]
I'm finding the [hardcoded axios timeout](https://github.com/searchkit/searchkit/blob/f59f28398928c47ef0ec3b25706c224b008766b5/src/core/transport/AxiosESTransport.ts#L13) too low for my environment, it'd be nice to be able to configure this.

**Comment by ssetem** [comment]
yep, for now you can require("AxiosESTransport") from "searchkit" and set the static field

**Comment by gaving** [comment]
@ssetem Ahhh awesome, thanks!

**Comment by alx** [comment]
Just in case someone needs a copy/paste:

```
// Override default 5s timeout with 20s timeout
import {AxiosESTransport} from "searchkit";
AxiosESTransport.timeout = 20000;
```

**Comment by alx** [comment]
Just in case someone needs a copy/paste:

```
// Override default 5s timeout with 20s timeout
import {AxiosESTransport} from "searchkit";
AxiosESTransport.timeout = 20000;
```

**Comment by joemcelroy** [comment]
fixed in #396 once pulled in. Closing issue
